apis/v1alpha1/collector_webhook.go

```diff
@@ -28,12 +29,41 @@ import (
 
 	"github.com/open-telemetry/opentelemetry-operator/internal/config"
 	ta "github.com/open-telemetry/opentelemetry-operator/internal/manifests/targetallocator/adapters"
+	"github.com/open-telemetry/opentelemetry-operator/internal/rbac"
 	"github.com/open-telemetry/opentelemetry-operator/pkg/featuregate"
 )
 
 var (
 	_ admission.CustomValidator = &CollectorWebhook{}
 	_ admission.CustomDefaulter = &CollectorWebhook{}
+
+	// targetAllocatorCRPolicyRules are the policy rules required for the CR functionality.
+	targetAllocatorCRPolicyRules = []*rbacv1.PolicyRule{
```

Comment:
Thanks @jaronoff97 for updating this. It still seems to be missing the namespace (list/watch) permissions in this list. Will that be added in too?

Reply:
ah i was going to let you add that in your pr, as your PR is going to be the first thing to need that i believe

Reply:
got it, i can sure add that. Thanks!